backend/app/main.py

```
@@ -0,0 +1,126 @@
+    app = FastAPI(
+        title="n-gage Backend",
+        description="Backend API for n-gage EdTech simulation platform",
+        version="2.0.0",
+        lifespan=lifespan,
+        docs_url="/docs",
+        redoc_url="/redoc",
+    )
+
+    # 0. Global exception handlers
+    @app.exception_handler(Exception)
+    async def global_exception_handler(request: Request, exc: Exception):
```

Ruff (0.14.8)

Unused function argument: `request`

(ARG001)